src/content/docs/en/guides/authentication.mdx

```diff
@@ -188,6 +188,78 @@ const session = await getSession(Astro.request);
 - [`auth-astro` on GitHub](https://github.com/nowaythatworked/auth-astro?tab=readme-ov-file#auth-astro)
 - [Auth.js documentation](https://authjs.dev/)
 
+## Clerk
+
+Clerk is a complete suite of embeddable UIs, flexible APIs, and admin dashboards to authenticate and manage your users. An [official SDK](https://clerk.com/docs/references/astro/overview) for Astro is available.
```

Comment:
```suggestion
Clerk is a complete suite of embeddable UIs, flexible APIs, and admin dashboards to authenticate and manage your users. An [official Clerk SDK for Astro](https://clerk.com/docs/references/astro/overview) is available.
```
link text should be descriptive for assistive technology, so just moving the description into the link!